### DISCUSSION TITLE
Meshoptimizer decompression for Unity not available! :(

### DISCUSSION THREAD

**Discussion by RuiBEAT** [atteneder/glTFast#311] [Q&A] [answered]
Hi,

Among the packages available for GLTFast is support for the EXT_meshopt_compression extension. However, [the link provided](https://docs.unity3d.com/Packages/com.unity.meshopt.decompress@0.1/manual/index.html) says that it is available through the Package Manager as preview package in the Unity registry. I have verified that this is not the case, and the package is not available as a preview package at least for Unity 2020.3 and Unity 2021.2. I've even tried to include it by adding its id in the manifest.json (as `"com.unity.meshopt.decompress": "0.1"`), but it doesn't resolve correctly.

This is a very interesting feature for us and we wanted to include it in our project. However, at the moment apparently nobody can benefit from its use because it is not available in any way :(

Please, any clarification on this? 

Regards,

**Comment by atteneder** [comment, marked as answer]
Hi @RuiBEAT ,

Thanks for pointing it out. I can confirm that following this generic instructions is flawed, since the package is not visible in the UPM.

Try adding it to the `manifest.json` under `dependencies` with an exact version like this:

```JSON
"com.unity.meshopt.decompress": "0.1.0-preview.4",
```

Let me know if that worked out.

**Reply by RuiBEAT** [reply]
Hi Andreas,

I haven't tested the meshopt feature yet, but with that version manually included in the manifest.json, the package resolves correctly.

Thank you!
